index.html

```diff
@@ -5502,7 +5504,101 @@ <h2>Base Context</h2>
   }
 }
 </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
+        <p>
+The verifiable credential and verifiable presentation data models leverage a
+variety of underlying technologies including [JSON-LD] and [JSON-SCHEMA-2018].
+This section will provide a comparison of the <code>@context</code>,
+<code>type</code>, and <code>credentialSchema</code> properties, and cover some
+of the more specific use cases where it is possible to use these features of the
+data model.
+        </p>
+        
+        <p>
+The primary purpose of the <code>type</code> property is to uniquely
+identify the type of the <code>verifiableCredential</code> in which it
+appears i.e. to indicate which set of claims the
+<code>verifiableCredential</code> contains. This property is mandatory and
+the value "VerifiableCredential" must appear in the set of values. Whilst
+it is good practice to include one additional value depicting the unique
+subtype of this verifiableCredential, it is not forbidden to either omit
+additional type values or to include multiple type values in the array.
+Since many verifiers will request verifiableCredentials of a specific
+subtype, then omitting the subtype value may make it more difficult for
+verifiers to inform the holder which verifiableCredential they require.
```

Comment:
> verifiableCredential

Strange that this isn't two words or not encapsulated in a `<code>` span. If you mean verifiableCredential the concept, then it should be `<a>verifiable credential</a>`. If you mean verifiableCredential the property, then it should be `<code>verifiableCredential</code>`.

Reply:
👍 I missed this one when I was going back through looking for all of these instances. Will update it

Reply:
There appear to be many other instances of `verifiableCredential` and `VerifiableCredential` which need similar attention.

Reply:
I'll do a once over to check all the type and verifiableCredential instances to make sure they get added. Going back and forth between the google document made it easy to drop and/or miss these when I was quickly just copying and pasting text over.

Reply:
I've gone and replace nearly every instance of `verifiableCredential` and `verifiable presentation` with `<a>verifiable credential</a>` or `<a>verifiable presentation</a>`. The one instance I kept in `<code>` tags was the one where we're stating the specific string value that's mandatory in the type array which I made `<code>VerifiableCredential</code>` to align with how it would be stated above. I'm fairly certain I've addressed this concern across the entire PR now.

Reply:
@TallTed I've also systematically gone through all of your Gdoc suggestions and made sure to get them into the PR now as well and make sure every change either made it in or some modification was made that aligned with your suggestions. We should be able to consolidate all review to the PR now with confidence that nothing has been left out at this point.